applications/registration-api/src/main/java/no/dcat/controller/ApiRegistrationPublicController.java

```diff
@@ -0,0 +1,53 @@
+package no.dcat.controller;
+
+import no.dcat.model.ApiRegistration;
+import no.dcat.service.ApiRegistrationRepository;
+import no.dcat.webutils.exceptions.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@RestController
+@RequestMapping(value = "/catalogs")
+public class ApiRegistrationPublicController {
+
+    private ApiRegistrationRepository apiRegistrationRepository;
+
+    @Autowired
+    public ApiRegistrationPublicController(
+        ApiRegistrationRepository apiRegistrationRepository
+    ) {
+        this.apiRegistrationRepository = apiRegistrationRepository;
+    }
+
+    /**
+     * Get all published api registration
+     *
+     * @return complete apiRegistration
+     */
+    @CrossOrigin
+    @RequestMapping(
```

Comment:
jeg tror vi må også tilla uautoriserte brukere i konf, for di api-cat skal ikke autorisere selv.